devices/patches/rcc/h7_usbotghs.yaml

```diff
@@ -0,0 +1,10 @@
+AHB1LPENR,C1_AHB1LPENR:
+  _modify:
+    USB1OTGLPEN,USB1OTGHSLPEN: #TODO: Help
+      name: USB1OTGLPEN
+    USB1OTGULPILPEN,USB1ULPILPEN,USB1OTGHSULPILPEN: #TODO: Help
+      name: USB1OTGULPILPEN
```

Comment:
@burrbull some of the devices have some, but not all, of the fields correct. Is there a better way to do this?

Reply:
```yaml
    "?~USB1OTGHSLPEN":
      name: USB1OTGLPEN
    "?~USB1ULPILPEN,USB1OTGHSULPILPEN":
      name: USB1OTGULPILPEN
```
Starting with `?~` means "if present".
If I understood your question right.